cmd/utils/flags.go

```diff
@@ -62,6 +62,21 @@ var (
 			EnvVars: []string{"DB_DBNAME"},
 		},
 	}
+	// s3 flags
+	S3Flags = []cli.Flag{
```

Comment:
I think we make variable name of flags to package name.

ServicesFlags

Reply:
Oh, I am sorry. Actually, I missed code convention here.
I have updated!